packages/devtools-components/src/tests/tree.js

```diff
@@ -653,14 +778,27 @@ function getTreeNodes(wrapper) {
   return wrapper.find(".tree-node");
 }
 
-function simulateKeyDown(wrapper, key) {
+function simulateKeyDown(wrapper, key, options) {
   wrapper.simulate("keydown", {
     key,
     preventDefault: () => {},
-    stopPropagation: () => {}
+    stopPropagation: () => {},
+    ...options
   });
 }
 
+function renderItemWithFocusableContent(x, depth, focused, arrow) {
+  let children = [{}, arrow, focused ? "[" : null, x, focused ? "]" : null];
+  if (x === "L") {
+    children = [
+      ...children,
+      dom.a({ id: "active-anchor", href: "#" }, "anchor")
```

Comment:
nit: could we put a space before the "anchor" ? It's only to make it a tad better in snapshots.